sections/security/non-root-user.md

```diff
@@ -0,0 +1,30 @@
+# Run Node as Noon-Root User
+
+### One Paragraph Explainer
+According to the 'Principle of least privilege' a user/process must be able to access only the information and resources that are necessary. Granting root access to an attacker opens a whole new world of malicious ideas like routing traffic to other servers. In practice, most Node apps don't need a root access and don't run with such privilege. However, there are two common scenario that might push to root usage: (1) to gain access to privilege port (e.g. port 80) Nodejs must run as root (2) Docker containers by default run as root(!). It's recommended for Node webserver to listen to non-privilege ports and rely on a reverse-proxy like nginx redirect from port 80. When building a docker image, highly secured apps should run the container with an alternate non-root user. Most docker clusters (e.g. Swarm, Kubernetes) allow setting the security context declaratively 
+
+### Code example - Building a Docker image as noon-root
+```javascript
+FROM node:some-version-here
```

Comment:
I'd just specify a version instead of the placeholder to make it more readable

Reply:
I'd agree to @lirantal, even the 'latest' tag would suffice for this, the rest seems pretty good though!

Reply:
@lirantal @BrunoScheufler good point